invenio_app_ils/permissions.py

```diff
@@ -59,3 +61,26 @@ def views_permissions_factory(action):
         return backoffice_permission()
     else:
         return deny_all()
+
+
+def filter_circulation_status(circulation_status):
```

Comment:
Is this an appropriate place to put this filtering function or would it make more sense to put it somewhere else?